src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -2,18 +2,49 @@
 
 import core.basesyntax.Storage;
 
+@SuppressWarnings("unchecked")
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_ITEMS_NUMBER = 10;
+    private Pair<K, V>[] pairs = new Pair[MAX_ITEMS_NUMBER];
```

Comment:
```suggestion
    private Pair<K, V>[] pairs;
```
move the initialization to the constructor
and use diamond new Pair<>[]